examples/oft-solana/tasks/solana/multisig.ts

```diff
@@ -27,3 +30,72 @@ export const createMintAuthorityMultisig = async (
         tokenProgramId
     )
 }
+
+/**
+ * Decode the signers of a multisig account and check if the expected signers
+ * are present and the quorum is 1/N.
+ * @param connection {Connection}
+ * @param multisigAddress {PublicKey}
+ * @param expectedSigners {PublicKey[]} the expected signers
+ */
+export const checkMultisigSigners = async (
+    connection: Connection,
+    multisigAddress: PublicKey,
+    expectedSigners: PublicKey[]
+) => {
+    // Fetch account info for the multisig.  "undefined" is treated as an error.
+    const accountInfo = await backOff(
+        async () => {
+            const accountInfo = await connection.getAccountInfo(multisigAddress)
+            if (!accountInfo) {
+                throw new Error('Multisig account not found')
+            }
+            return accountInfo
+        },
+        {
+            numOfAttempts: 10,
+            startingDelay: 5000,
```

Comment:
I'd add some maxDelay here because with exponential backoff and 5s initial delay there's gonna be a long wait in the worst case

Reply:
Fair point.  Unlikely to happen, but I could see it if the RPC is flaky.  I have included in a follow-up.